exonum/src/runtime/mod.rs

```diff
@@ -660,32 +660,32 @@ impl<T: WellKnownRuntime> From<T> for RuntimeInstance {
 }
 
 /// Instance descriptor contains information to access the running service instance.
-#[derive(Debug, Copy, Clone, PartialEq, Eq)]
-pub struct InstanceDescriptor<'a> {
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct InstanceDescriptor {
```

Comment:
I think we've agreed that `InstanceDescriptor` should remain as is 🙃 I'm not principally against this change though, it can simplify some code. Does it impact performance?

Reply:
To be honest I haven't measured it. But I think it won't bring a significant regression.